package.json

```diff
@@ -75,7 +75,7 @@
     "secp256k1": "^3.7.1",
     "sinon": "^9.0.2",
     "solidity-docgen": "^0.5.3",
-    "truffle": "^5.1.14",
+    "truffle": "^5.1.33",
```

Comment:
bump version to deal with truffle issue we've been having.

Reply:
What issue?

Reply:
The error we were having with `TypeError: Cannot read property 'number' of null` ([issue](https://github.com/trufflesuite/truffle/issues/3130)) was resolved in [this](https://github.com/trufflesuite/provider-engine/pull/6) PR and was released to truffle in this 33 version :)

Reply:
Oh nice!